cadcutils/cadcutils/net/ws.py

```diff
@@ -767,6 +763,21 @@ def _resolve_destination_file(dest, src_md5, default_file_name):
         else:
             return final_dest, final_dest
 
+    def compute_file_md5(file_path):
+        """Compute the md5 hash of a file.
+        Args:
+            file_path (pathlib.Path): the path to the file.
+
+        Returns:
+            str: the md5 hash of the file.
+        """
+        md5_hash = hashlib.md5()
+        buffer_size = 50 * 1024
```

Comment:
For my understanding only, does the larger buffer size mean that calling apps will need access to that much memory?

Reply:
Yes. It's a tradeoff. Larger buffer speeds up the calculations but increases the memory footprint of the application. I think 50k is a reasonable compromise unless you have a better suggestion.